docker.go

```diff
@@ -89,6 +90,28 @@ func getIPsToNames(client dockerClienter, id string) (ipsToNamesMap, error) {
 			names = appendNames(names, name)
 		}
 
+		if label, ok := containerFull.Config.Labels["com.costela.docker-etchosts.add_hosts"]; ok {
+			if (strings.HasPrefix(label, "[")) {
+				var parsed []string; 
+				err := json.Unmarshal([]byte(label), &parsed)
+				if err != nil {
+					log.Errorf("error parsing JSON: %s", err)
+				}
+				names = append(names, parsed...)
+			} else if (strings.HasPrefix(label, "\"")) {
+				var parsed string; 
+				err := json.Unmarshal([]byte(label), &parsed)
+				if err != nil {
+					log.Errorf("error parsing JSON: %s", err)
+				}
+				names = append(names, parsed)
+			} else if (strings.HasPrefix(label, "{")) {
+				log.Errorf("JSON objects are not supported: %s", label)
+			} else {
+				names = append(names, label)
+			}
```

Comment:
Maybe we can also be a bit more defensive here?
We could check in the first condition against a valid domain regex (something optimistic like `[a-zA-Z][a-zA-Z0-9.-]*` - from hosts(5) -  will probably work for 90% of cases) and error for everything else?
If you think the regexp is too restrictive, we could at least filter out things that would break the `/etc/hosts` file, like newlines, just to be on the safe side.
WDYT?

Reply:
I made it a little bit stricter based on the manual. Thanks!